chapters/generative-models.md

```diff
@@ -272,8 +284,8 @@ The probability of an event $$A$$ (such as the above program returning `[true, f
 A **probability distribution** is the probability of each possible outcome of an event. For instance, we can examine the probability distribution on values that can be returned by the above program by sampling many times and examining the histogram of return values:
 
 ~~~~
-var randomPair = function () { return [flip(), flip()]; };
-viz.hist(repeat(1000, randomPair), 'return values');
+var randomPair = function () {  [flip(), flip()] }
```

Comment:
I think the `return` is actually needed by webppl to compile correctly? Let's keep it to be safe..